src/components/OptionComponent/index.js

```diff
@@ -133,6 +134,7 @@ const OptionComponent = props => {
 
   useEffect(() => {
     setShowDescription(false)
+    setMarginToken(tokens[1])
   }, [chainId, mode])
```

Comment:
[Check dependency array](https://devtrium.com/posts/dependency-arrays#what-should-be-put-in-the-dependency-array)